File: Lab4/pliki/zad2/Supplier.cs
namespace MateuszLopacinskiEFProducts
{
  internal class Supplier
  {
    public int SupplierID { get; set; }
    public string CompanyName { get; set; }
    public string Street { get; set; }
    public string City { get; set; }

    public override string ToString()
    {
      return CompanyName;
    }
  }
}

File: Lab5/2/Supplier.java
package com.matipl01;

import javax.persistence.*;

@Entity
public class Supplier {
    @Id
    @GeneratedValue(strategy = GenerationType.AUTO)
    public int supplierID;

    private String companyName;
    private String street;
    private String city;

    public Supplier() {}

    public Supplier(String companyName, String street, String city) {
        this.companyName = companyName;
        this.street = street;
        this.city = city;
    }

    @Override
    public String toString() {
        return companyName;
    }
}
